Pilot/pilotCommands.py

```diff
@@ -484,27 +474,21 @@ def execute(self):
 
     # If NumberOfProcessors or MaxRAM are defined in the resource configuration, these
     # values are preferred
+
+    # pilotProcessors is basically the number of processors this pilot is "managing"
+    self.pp.pilotProcessors = numberOfProcessorsOnWN
```

Comment:
Incidentally, this naming is wrong for historical reasons: the numberOfProcessorsOnWN is actually receiving the number of processors allocated to the job slot.